index.rst

```diff
@@ -2,7 +2,19 @@ Raiden Network Specification
 =================================
 
 
-This is a tentative specification for the `Raiden Network <http://raiden.network/>`_. It's under development and will be further refined in subsequent iterations.
+This is the specification of the `Raiden Network <http://raiden.network/>`_.
+
+Its goal is to provide an exact description of the Raiden protocol. So the structure of
+a mediated transfer, message formats and such are to be found here, while implementation
```

Comment:
What does `and such` mean in this case? do we have anything else that describes the protocol other than messages?